lib/buffer.js

```diff
@@ -549,33 +549,10 @@ Buffer.prototype.toJSON = function() {
 };
 
 
-// TODO(trevnorris): currently works like Array.prototype.slice(), which
-// doesn't follow the new standard for throwing on out of range indexes.
```

Comment:
@trevnorris That's out of scope of this PR =).
Btw, in that CI even the linter blew up on the Jenkins side.